db/migrate/20220331003145_port_attr_encrypted_to_encrypts.rb

```diff
@@ -0,0 +1,63 @@
+class PortAttrEncryptedToEncrypts < ActiveRecord::Migration[7.0]
```

Comment:
Make this a rake task so that migrations don't depend on attr_encrypted being installed forever

Reply:
I've moved the code into a rake task and remove the extra migrations